src/commonMain/kotlin/com/ashampoo/kim/format/jpeg/iptc/IptcMetadata.kt

```diff
@@ -38,4 +38,9 @@ data class IptcMetadata(
 
         return sb.toString()
     }
+
+    operator fun plus(other: IptcMetadata): IptcMetadata = IptcMetadata(
+        records = records + other.records,
+        rawBlocks = rawBlocks + other.rawBlocks
+    )
```

Comment:
Thank you for working on the TODO to merge the records.

Unfortunately this is not the correct way to do that and that's why the unit tests fail.

Some photos, in our test set photo_22.jpg, have multiple IPTC records. In this particular case they are not distinct, but repeated, which would result in illegal multiple `creation date` fields.

On the other hand there are fields like `keywords` that can be merged this way.

I left this topic unresolved (like Apache Commons Imaging does), because I did not see a file with valid use of multiple IPTC fields yet and it requires further work to decide on how to merge the different fields.

Can you remove this code from your PR?